Ships as Raw

```diff
@@ -0,0 +1,860 @@
+FrontArcOffset|2
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|14.3
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|47
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|53
+//How far from the rear of the ship do the rear hull zone lines meet//
+//This ship has a 2 section between the rear and front conjuctions.//
+
+
+Name|MC80 Assault Cruiser
+Type|MC80 Assault Cruiser
+Keywords|0
+Faction|Rebel
+Size|Large
+Points|114
+Hull|8
+defenseTokens|Rd Br Rd Cn
+Command|3
+Squad|3
+Eng|4
+Speed1|1
+Speed2|11
+Speed3|0
+Speed4|0
+FrontShields|4
+SideShields|4
+RearShields|3
+FrontArmament|RRB
+SideArmament|RRRBB
+RearArmament|BRR
+AntiSquadron|BB
+Upgrade|Title Officers Defensive Defensive Support Team Ion Cannon Turbo Lasers 
+FrontArcOffset|0
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|0
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|64
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|63
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+Name|MC80 Command Cruiser
+Type|MC80 Assault Cruiser
+Keywords|0
+Faction|Rebel
+Size|Large
+Points|106
+Hull|8
+defenseTokens|Rd Br Rd Cn
+Command|3
+Squad|4
+Eng|4
+Speed1|1
+Speed2|11
+Speed3|0
+Speed4|0
+FrontShields|4
+SideShields|4
+RearShields|3
+FrontArmament|RBB
+SideArmament|BBBRRR
+RearArmament|BBR
+AntiSquadron|BB
+Upgrade|Title Officers Defensive Offensive Support Team Ion Cannon Turbo Lasers 
+FrontArcOffset|0
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|0
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|64
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|63
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+
+Name|MC75 Armored Cruiser
+Type|MC75 Armored Cruiser
+Keywords|0
+Faction|Rebel
+Size|Large
+Points|104
+Hull|9
+defenseTokens|Rd Br Cn Cn
+Command|3
+Squad|3
+Eng|4
+Speed1|2
+Speed2|11
+Speed3|011
+Speed4|0
+FrontShields|4
+SideShields|3
+RearShields|3
+FrontArmament|BBBKK
+SideArmament|BBRRR
+RearArmament|BK
+AntiSquadron|BK
+Upgrade|Title Officers Defensive Offensive Support Team Ion Cannon Turbo Laser Weapons team Ordance   
+FrontArcOffset|20
+//How far from the front corner does the hull zone line hit the cardboard edge//
+RearArcOffset|0
+//How far from the rear corner does the hull zone line hit the cardboard edge//
+FrontArcConjunction|65
+//How far from the front of the ship do the front hull zone lines meet//
+RearArcConjunction|62
+//How far from the rear of the ship do the rear hull zone lines meet//
+
+
+Name|MC75 Ordnance Cruiser
+Type|MC75 Ordnance Cruiser
+Keywords|0

```

Comment:
What was the rationale on the 0 vs None again?  I def did None on the squadrons, which isn't to say that it is better than 0.

Mostly a fan of consistancy.

Reply:
No problem I didn't think of making them the same I get it